escalation/ticketEscalation.sw.yaml

```diff
@@ -11,7 +11,7 @@ start: CreateTicket
 extensions:
   - extensionid: workflow-uri-definitions
     definitions:
-      notifications: "https://raw.githubusercontent.com/janus-idp/backstage-plugins/%40janus-idp/plugin-notifications-backend%401.0.3/plugins/notifications-backend/src/openapi.yaml"
+      notifications: ../shared/specs/notifications-openapi.yaml
```

Comment:
i don't know. how you build the image. if the repo is added to the image then it will work. if you have a better solution please share

Reply:
@masayag WDYT?

Reply:
@gabriel-farache do you have any experience with referring to a relative resource?

Reply:
We used to (briefly) have a shared folder for specs in the examples repo but we moved to use URL instead so I guess it can be built locally but I am not sure about build in docker, we can try to run a CI on this branch and see if it works

Or @ydayagi you can use the makfile I guess (see https://github.com/parodos-dev/serverless-workflows/blob/main/make.md)

Reply:
The build in the CI seems to work even if the e2e test fails because backstage fails to execute the workflow:
`Error when executing workflow: TypeError: fetch failed `